src/index.tsx

```diff
@@ -160,12 +177,9 @@ type CommonProps = {
    */
   animatedPosition?: Animated.Value<number>;
   /**
-   * Configuration for the timing reanimated function
+   * Configuration for the timing or spring reanimated function
    */
-  animationConfig?: {
-    duration?: number;
-    easing?: Animated.EasingFunction;
-  };
+  animationConfig?: Animated.SpringConfig & Animated.TimingConfig;
```

Comment:
`animationConfig` prop definition can be removed as well from `CommonProps`, it's overridden and documented further down in lines 214 and 222